### DISCUSSION TITLE
Locked folders linked to a sponsor tier?

### DISCUSSION THREAD

**Discussion by oldcityws** [community/community#59454] [Other Feature Feedback, Questions, & Ideas]
**Product Feedback**

We should be able to directly link a locked folder AND a specific Discussion area to a sponsorship tier.
As soon as someone pays for the linked tier, then they should have access without the developer having to manually grant access or do other work.
I think that this would make it much more efficient to streamline provision of benefits for sponsors.

**Comment by anjuan** [comment]
Hi, @oldcityws, thanks for the feedback. If I understand you correctly, you want a way for sponsors at a specific tier to get private access to a locked folder and also to a private discussion? If that' correct, you can currently [attach a private repository](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/managing-your-sponsorship-tiers#adding-a-repository-to-a-sponsorship-tier) to a tier that only sponsorships at that tier can access. This doesn't address your request for a private discussion, but it does provide a way for controlled access to a folder (in a repository).

**Reply by oldcityws** [reply]
My issue is that my repos are owned by my account... kind of strange that it requires ownership by an organization before I can link it to a sponsorship tier.  That seems really convoluted.

**Reply by anjuan** [reply]
While we won't touch how repos are attached to tiers any time soon, I will keep your suggestion of allowing ownership by individual accounts in mind if we decide to make a change.

**Reply by oldcityws** [reply]
What would the decision-makers need to see - or have - in order to see the change as one worthy of being made?

**Reply by anjuan** [reply]
Thank you, but I don't think we need more information. The Sponsors team is always refining our roadmap, and we have a list of things to work on based on impact and priority.
